packages/core/src/i18n/arrayTranslations.ts

```diff
@@ -0,0 +1,40 @@
+export interface DefaultTranslation {
+    key: ArrayTranslationEnum,
+    default: (variable?: string) => string
+}
+
+export enum ArrayTranslationEnum{
+    addTooltip = 'addTooltip',
+    addAriaLabel = 'addAriaLabel',
+    removeTooltip = 'removeTooltip',
+    upAriaLabel = 'upAriaLabel',
+    downAriaLabel = 'downAriaLabel',
+    noSelection = 'noSelection',
+    removeAriaLabel = 'removeAriaLabel',
+    noDataMessage = 'noDataMessage',
+    deleteDialogHeader = 'deleteDialogHeader',
+    deleteDialogMsg = 'deleteDialogMsg',
+    deleteDialogAccept = 'deleteDialogAccept',
+    deleteDialogDecline = 'deleteDialogDecline'
+}
+
+export type ArrayTranslations = {
+    [key in ArrayTranslationEnum]?: string
+}
+
+export const arrayDefaultTranslations: DefaultTranslation[] = [
+    {key: ArrayTranslationEnum.addTooltip, default: (input) => input?`Add to ${input}`:'Add'},
+    {key: ArrayTranslationEnum.addAriaLabel, default: (input) => input?`Add to ${input}`:'Add'},
+    {key: ArrayTranslationEnum.removeTooltip, default: () => 'Delete'},
+    {key: ArrayTranslationEnum.upAriaLabel, default: (input) => `Move ${input} up`},
+    {key: ArrayTranslationEnum.downAriaLabel, default: (input) => `Move ${input} down`},
+    {key: ArrayTranslationEnum.removeAriaLabel, default: () => 'Delete button'},
+    {key: ArrayTranslationEnum.noDataMessage, default: () => 'No Data'},
+    {key: ArrayTranslationEnum.noSelection, default: () => 'No Selection'},
+    {key: ArrayTranslationEnum.deleteDialogHeader, default: () => 'Confirm Deletion'},
+    {key: ArrayTranslationEnum.deleteDialogMsg, default: () => 'Are you sure you want to delete the selected entry?'},
+    {key: ArrayTranslationEnum.deleteDialogAccept, default: () => 'YES'},
+    {key: ArrayTranslationEnum.deleteDialogDecline, default: () => 'NO'}
```

Comment:
Why are these translations in all caps? As I saw it in the changes in `DeleteDialog.tsx` this was not the case before. I think we should stay with the old capitalization, i.e. `Yes` and `No`.